windows-installer/docs/INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# Certbot for Window (beta)
+
+The Certbot development team is proud to propose you the first beta release of Certbot for Windows.
+
+This document explains how to install Certbot and use it on Windows.
+
```

Comment:
I would add (since some users on the forum are often mixed up about the difference between their local OS and their remote server OS) something like:

> Please note that this option is intended for the situation where your *web server* runs Windows. If you use Windows on your personal computer but have a web server with a different operating system, you should normally install Certbot on the web server (probably via SSH) instead of on your local computer. This document is not intended to describe that situation.